charters/MediaStrategyCharter.md

```diff
@@ -0,0 +1,42 @@
+# Jupyter Media Strategy Working Group
+
+## Purpose
+
+The mission of the Jupyter Media Strategy Working Group (JMS) is to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter. The Jupyter[ governance model ](https://jupyter.org/governance/standing_committees_and_working_groups.html)includes a distributed model of responsibilities. This means that by design Jupyter publications, social media posts, promotions and other media activity are meant to be community driven. The JMS is vendor neutral and will provide mechanisms to enable the community to speak publicly through the official Jupyter communication channels. The JMS will also help to improve Jupyter media activity by creating strategy and guidelines, serving as editors for existing media channels, and overseeing creation/delegation of new media channels.
+
+
+# Editorial Principles
+
+The JMS makes editorial decisions related to whether or not to accept/post content on our official channels based on the following principles:
+
+- Prioritize content about the work of Project Jupyter and its subprojects and is beneficial to the Jupyter community.
+- Ensure that any third party content is beneficial to the Jupyter community.
+- Content may be written in any language, provided it comes from or has been vetted by a member of the Jupyter Union of Councils.
+- The Jupyter blog is the primary channel with which Jupyter communicates with the community.
+- Social media channels will be used to amplify the reach of the blog. Posting on a particular platform is not an endorsement of that platform by Project Jupyter, but a means to communicate news to the Jupyter Community. With the exception of our preferred channel, we will use these channels to broadcast messages only, and will therefore not engage in two-way communication with users on said channels.
```

Comment:
> we will use these channels to broadcast messages only, and will therefore not engage in two-way communication with users on said channels.

Does that preclude a later group to engage with users and reply to their questions on non-prefered channel ? Even if it's replying "this channel is not used for support, please reach out via <platform>" which is  technically a two way communication.

Reply:
This relates only to official public-facing official Jupyter accounts.  The user profile will make it clear that it is considered broadcast only.  Individual users in the Jupyter community are still free to interact on these channels.